anyric/super-record
# #168433335 Add views for the Expenses App

#### What does this PR do?
This PR add views for accessing and manipulating the expenses app  data from the browser.

#### Description of Task to be completed?
- Add views for the expenses app.

#### How should this be manually tested?
- Clone this branch
- In the root directory run `python manage.py runserver`
- Navigate to `http://127.0.0.1:8000/` to login
- Click on the `Expense` menu from the dashboard
- Fill the expense form to register new expenses incurred
- Select the `Action` links to edit or delete an expense
- Navigate to `http://127.0.0.1:8000/accounts/settings` 
- Select `Expenses Category` menu on the left to see list of expense category
- Click `New Category` menu on the top right to add new expense category
- Select the `Action` links to edit or delete an expense category
- Select `Expenses` menu on the left to see list of sales

#### Any background context you want to provide?
Currently the expenses app doesn't have a view function. There should be a view from which users can interact while adding and manipulating expenses record. This should enable access to the data through a graphical interface

#### What are the relevant pivotal tracker stories?
[#168433335](https://www.pivotaltracker.com/story/show/168433335)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29800415/64872330-acb7d500-d64f-11e9-8694-bd026d0d1b0d.png)

![image](https://user-images.githubusercontent.com/29800415/64855381-6d29c280-d628-11e9-8c97-d65d11c138d7.png)

![image](https://user-images.githubusercontent.com/29800415/64855365-669b4b00-d628-11e9-8f5b-38befb378513.png)

![image](https://user-images.githubusercontent.com/29800415/64855324-4f5c5d80-d628-11e9-9a6e-3d7619738413.png)

#### Questions:
N/A
